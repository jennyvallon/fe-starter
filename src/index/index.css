@import './variables.css';
@import './typography.css';

/* global styles */

body {
  margin: 0;
  padding: 0;
  font-family: sans-serif;
}
