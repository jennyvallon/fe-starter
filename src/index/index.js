import React from 'react';
import ReactDOM from 'react-dom';
import {Provider} from 'react-redux';
import './index.css';
import store from '../redux/store';
import Root from '../components/Root/index';
import registerServiceWorker from './registerServiceWorker';

ReactDOM.render(
    <Provider store={store}>
        <Root />
    </Provider>
    , document.getElementById('root'));
registerServiceWorker();





