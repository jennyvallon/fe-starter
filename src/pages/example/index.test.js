import React from 'react';
import {ExamplePage} from './index';

test('Render the Root component', () => {
    const component = shallow(
        <ExamplePage/>
    );
    expect(component).toMatchSnapshot();
});