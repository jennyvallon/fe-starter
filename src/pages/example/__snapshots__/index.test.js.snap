// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the Root component 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg"
    />
    <h1
      className="App-title"
    >
      Welcome to poot
    </h1>
  </header>
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      the is butternut
    </code>
     and save to reload.
  </p>
</div>
`;
